Update pom.xml JaCoCo version to 0.8.8

v-niranjan opened the pull request:
Updating JaCoCo version to support code build on [Automate Java container deployments with Azure Pipelines](https://docs.microsoft.com/en-us/learn/modules/deploy-java-containers/4-deploy-java-container )

v-niranjan committed:
Update pom.xml

Updating JaCoCo version to support code build on https://docs.microsoft.com/en-us/learn/modules/deploy-java-containers/4-deploy-java-container

v-niranjan commented:
Hi @juliakm , 

PR is ready for Technical review.

juliakm commented on pom.xml:
```diff
@@ -40,7 +40,7 @@
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.7.9</version>
+        <version>0.8.2</version>
```
Is there a reason why this version is better than some of the newer ones? https://mvnrepository.com/artifact/org.jacoco/jacoco-maven-plugin

v-niranjan replied:
Yes, v0.8.8 works fine with https://docs.microsoft.com/en-us/learn/modules/deploy-java-containers/
Tested on https://github.com/v-niranjan/mslearn-java-containers/pull/6

v-niranjan committed:
Update pom.xml

Yes, v0.8.8 works fine with https://docs.microsoft.com/en-us/learn/modules/deploy-java-containers/

juliakm reviewed (approved):
Thank you! This will help with future-proofing!

v-niranjan commented:
Hi @juliakm , 

I don't have access to merge this branch. Can you please merge it to the repo and as discussed kindly rename the default branch from master-->main 

Regards, 
@v-niranjan
